### PR TITLE
NDEV-18113: Update crd readme

### PR DESCRIPTION
**Pull request by VedRatan**

### CODE REVIEW

File: charts/enterprise-kyverno-operator/README.md [hunk 1]
```diff
@@ -2,6 +2,9 @@
 Nirmata Kyverno Opertor is a Kubernetes Operator to manage lifecycle of Kyverno, Adapters and Nirmata supported policies. 
 
 ## Prerequisites
+
+As a prerequisite, user needs to install [kyverno-operator-crd](../enterprise-kyverno-operator-crd/) to install operator.
```
Comment (anushkamittal2001) [inline charts/enterprise-kyverno-operator/README.md:6]: this is not a prerequisite
this is if the user wants to ensure delete of crds is done

File: charts/enterprise-kyverno-operator-crd/README.md [hunk 2]
```diff
@@ -24,212 +22,5 @@ Add the chart repository and install the chart
...
-Kyverno Operator uses webhooks to provide enhanced functionality such as logging user information in resource change events logged into the Kubernetes event stream, and some enhanced semantic checks for custom resources.
-
-Webhooks need a few SSL key-certificates to work properly. By default, kyverno operator manages the creation and rotation of these. But alternatively these can be managed manually too.
-
-###  Manual certificate management
-For this, one would need to manually create the secret containing certificate and keys needed by the operator webhook, and provide the base64 encoded CA bundle to helm install command below as well. The secret should be of type `kubernetes.io/tls`, with keys `tls.crt`, `tls.key`, and `ca.crt`. The value of `ca.crt` should also be provided to the helm install command (arg `--set certManager=other --set caBundle=...`) so that it gets added to the CABundle for needed webhook configurations.
+helm install nirmata-kyverno-operator-crd nirmata/nirmata-kyverno-operator-crd -n nirmata-system --create-namespace --set licenseKey=<licenseKey>[,apiKey=<api key>] [--version v0.3.0-rc.. if using release candidate]
```
Comment (anushkamittal2001) [inline charts/enterprise-kyverno-operator-crd/README.md:25]: can we show the entire workflow of installing both charts and uninstalling?

File: charts/enterprise-kyverno-operator-crd/Chart.yaml [hunk 3]
```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 name: nirmata-kyverno-operator-crd
 description: Helm Chart for Enterprise Kyverno Operator
 type: application
-version: v0.4.7
+version: v0.4.8
```
Comment (anushkamittal2001) [inline charts/enterprise-kyverno-operator-crd/Chart.yaml:5]: do we need to bump version for README.md changes?

File: charts/enterprise-kyverno-operator/README.md (hunk 1 above)
Comment (VedRatan) [reply charts/enterprise-kyverno-operator/README.md:6]: But we are moving the CRD inside the different chart so if a user go ahead by directly installing the operator chart, he/she will caught up in a error

Comment (anushkamittal2001) [reply charts/enterprise-kyverno-operator/README.md:6]: we didnt remove anything from the operator chart right?

if the user doesnt do this they will still have the old flow

Comment (VedRatan) [reply charts/enterprise-kyverno-operator/README.md:6]: we removed the CRD from operator chart and moved it into the separate chart

File: charts/enterprise-kyverno-operator-crd/Chart.yaml (hunk 3 above)
Comment (VedRatan) [reply charts/enterprise-kyverno-operator-crd/Chart.yaml:5]: lint tests were failing without bumping

File: charts/enterprise-kyverno-operator-crd/README.md (hunk 2 above)
Comment (VedRatan) [reply charts/enterprise-kyverno-operator-crd/README.md:25]: So do we need to omit it from prerequisites?

File: charts/enterprise-kyverno-operator/README.md (hunk 1 above)
Comment (anushkamittal2001) [reply charts/enterprise-kyverno-operator/README.md:6]: got it.

File: charts/enterprise-kyverno-operator-crd/Chart.yaml (hunk 3 above)
Comment (anushkamittal2001) [reply charts/enterprise-kyverno-operator-crd/Chart.yaml:5]: got it

**Review by anushkamittal2001 — APPROVED** [review]

**Pull request merged**